contrib/kafka/pkg/dispatcher/dispatcher_test.go

```diff
@@ -427,7 +426,7 @@ func TestSubscribeError(t *testing.T) {
 	}
 	err := d.subscribe(channelRef, subRef)
 	if err == nil {
-		t.Errorf("expected error want %s, got %s", "error creating consumer", err)
+		t.Errorf("Expected error want %s, got %s", "error creating consumer", err)
```

Comment:
errors should start with a lowercase letter

Reply:
Not in the tests :)